src/formatDate.js

```diff
@@ -8,7 +9,41 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const separador = fromFormat[3];
+  const separadorTo = toFormat[3];
+
+  const newDate = date.split(separador);
+
+  const dateObj = {};
+
+  for (let i = 0; i < fromFormat.length; i++) {
+    dateObj[fromFormat[i]] = newDate[i];
```

Comment:
Instead of using a for loop to create the `dateObj`, you can use array destructuring with the `fromFormat` to directly assign the values to the `dateObj` keys, which can make the code more concise and readable.